### PR TITLE
Add Models, Serializers and Views for Interests and Activities

### PR DESCRIPTION
**Pull request by Termuellinator**
I've created the models, serializers and paginated views for interests and activities, as well as a custom permission so only super users can create or modify those.

### CODE REVIEW

File: apps/timeplace/serializers.py
Comment (SteffenPyGo) [inline apps/timeplace/serializers.py:1]: Regarding the 'fields = "__all__"':

https://docs.djangoproject.com/en/4.2/topics/forms/modelforms/

>The alternative approach would be to **include all fields automatically,** or remove only some. This fundamental approach is known to be **much less secure** and has led to serious exploits on major websites

**Review by SteffenPyGo — APPROVED** [review]
Looks good to me. Thank you for your work.

File: apps/timeplace/serializers.py
Comment (Termuellinator) [reply apps/timeplace/serializers.py:1]: Good catch - the link to the blogpost in https://github.blog/2012-03-04-public-key-security-vulnerability-and-mitigation/ is unfortunately dead. I'll try to do some research on that an see if that is actually a problem with such "unimportant" and small models.

Comment (Termuellinator) [reply apps/timeplace/serializers.py:1]: https://archive.ph/NJyv should only affect frontend i think. Totally missed that you linked the forms-docs ^^

**Pull request merged**